resources/src/sessionCookie.php

```
@@ -15,6 +15,8 @@ class SessionCookie
     public function __construct()
     {
         $this->redirect = new Redirect;
+        $conf = new ProjectConfig;
+        $conf->disableError();
```

Find another alternative, hiding the error doesn't mean the issue is not there

---

its not an error na its a notice!

---

How about I add `session_destroy()` before session_start

---

Then you cannot find out who has logged in

---

@guptahemant  OOOps I also found another prob with menu Ill just make another fix and let you know

---

@AJV009, @guptahemant  I found it! The session warning is because we are creating an object of `SessionCookie` in `index.php` and also in `books.php` so wherever `book` object is created it shows an error. So yeah we can safely hide the warnings.

---

Nice, Now this means you need to modify the behaviour of session constructor, so that session_start is not invoked from it twice.

---

Refer this thread for possible solutions: https://stackoverflow.com/questions/6249707/check-if-php-session-has-already-started

---

> Refer this thread for possible solutions: https://stackoverflow.com/questions/6249707/check-if-php-session-has-already-started

got it